docker-entrypoint.sh

```diff
@@ -22,21 +25,18 @@ echo "### openrouteservice configuration ###"
 # if Tomcat built before, copy the mounted ors-config.json to the Tomcat webapp ors-config.json, else copy it from the source
 if [ -d "/usr/local/tomcat/webapps/ors" ]; then
   echo "Tomcat already built: Copying /ors-conf/ors-config.json to tomcat webapp folder"
-	cp -f /ors-conf/ors-config.json $tomcat_ors_config
+	cp -f ${public_ors_config} ${tomcat_ors_config}
 else
-	if [ ! -f /ors-conf/ors-config.json ]; then
-	  echo "No ors-config.json in ors-conf folder. Copy config from ${source_ors_config}"
-		cp -f $source_ors_config /ors-conf/ors-config.json
+	if [ ! -f $public_ors_config ]; then
+	  echo "No ors-config.json in ors-conf folder. Copying config from ${source_ors_config}"
+		cp -f ${source_ors_config} ${public_ors_config}
 	else
-	  echo "ors-config.json exists in ors-conf folder. Copy config to ${source_ors_config}"
-		cp -f /ors-conf/ors-config.json $source_ors_config
+	  echo "ors-config.json exists in ors-conf folder. Copying config to ${source_ors_config}"
+		cp -f ${public_ors_config} ${source_ors_config}
 	fi
 	echo "### Package openrouteservice and deploy to Tomcat ###"
-	mvn -q -f /ors-core/openrouteservice/pom.xml package -DskipTests && \
-	cp -f /ors-core/openrouteservice/target/*.war /usr/local/tomcat/webapps/ors.war
+	cp -f /ors-core/ors.war /usr/local/tomcat/webapps/ors.war
 fi
 
-/usr/local/tomcat/bin/catalina.sh run
-
-# Keep docker running easy
-exec "$@"
+# so docker can stop the process gracefully
+exec /usr/local/tomcat/bin/catalina.sh run
```

Comment:
Without exec'ing here, docker takes 10 secs to stop a container, since it'd do it in another shell.